### ISSUE TITLE
TediCross 0.9.4 Sent Stickers are always logged to Console

### ISSUE THREAD

**Issue by tux93** [TediCross/TediCross#118]
After updating to 0.9.4 every time someone sends a sticker the stickers properties are output to the console like this:
```
{ width: 512,
  height: 512,
  emoji: '😴',
  set_name: 'derideal',
  is_animated: false,
  thumb:
   { file_id: 'AAQBAAMsAAOz4UcCReCNb3FopM4Q9AowAAQBAAdtAANkWAACFgQ',
     file_size: 6500,
     width: 128,
     height: 128 },
  file_id: 'CAADAQADLAADs-FHAkXgjW9xaKTOFgQ',
  file_size: 37976 }
```

**Comment by Suppen** [comment]
I forgot to remove a line used for debugging. It is harmless, and will be removed in the next update

**Comment by tux93** [comment]
Closing it since it was fixed in 0.9.5
